pkgs/applications/misc/translatelocally/default.nix

```
@@ -26,6 +27,11 @@ in stdenv.mkDerivation (finalAttrs: {
       3rd_party/bergamot-translator/3rd_party/marian-dev/src/common/git_revision.h
   '';
 
+  # https://github.com/XapaJIaMnu/translateLocally/blob/81ed8b9/.github/workflows/build.yml#L330
+  postConfigure = lib.optionalString stdenv.isAarch64 ''
+    bash /build/source/cmake/fix_ruy_build.sh /build/source /build/source/build
```

Does not make any assumptions about `$NIX_BUILD_TOP` and `src.name`

```suggestion
    bash ./cmake/fix_ruy_build.sh . ./build
```